### DISCUSSION TITLE
discord.js/src/managers/GuildMemberRoleManager error idk

### DISCUSSION THREAD

**Discussion by tontonharibo** [discordjs/discord.js#9237] [Q&A]
### Which package is this bug report for?

discord.js

### Issue description

Hello, 
First issue, sorry if it is not complete. 

I don't know how to explain this error, see screen 

I don't know how it happened, I restarted my bot and suddenly it didn't work anymore and had this error. 
It only happens on one server, I tested on others, it works without problem. 

Version d.js = 14.3.0 
![image](https://user-images.githubusercontent.com/52158732/225593307-1f8b3812-7d83-4adf-a4e4-4de0ecfc66f6.png)

### Package version

14.3.0

### Node.js version

v18.3.0

### Priority this issue should have

Low (slightly annoying)

### Which partials do you have configured?

GuildMember, Message

### Which gateway intents are you subscribing to?

Guilds, GuildMembers, GuildModeration, GuildEmojisAndStickers, GuildIntegrations, GuildWebhooks, GuildInvites, GuildVoiceStates, GuildPresences, GuildMessages, GuildMessageReactions, GuildMessageTyping, DirectMessages, DirectMessageReactions, DirectMessageTyping, MessageContent, GuildScheduledEvents, AutoModerationConfiguration, AutoModerationExecution

**Comment by jaw0r3k** [comment]
have you modified roles cache?

**Reply by tontonharibo** [reply]
No, but how could I do that ?

**Reply by jaw0r3k** [reply]
you shouldnt

**Reply by tontonharibo** [reply]
Okay, so
Do you have any idea how to correct this error?

**Reply by jaw0r3k** [reply]
can you console your guild.roles.cache? and check if it has your every guild role

**Reply by tontonharibo** [reply]
I don't know if there are all the roles, there are quite a few, (especially with the roles of all the bots I have.) but yes, it looks good

**Comment by tontonharibo** [comment]
Can anyone help me plz?

**Reply by tontonharibo** [reply]
Is my guild screwed up or can I fix this error?
